modules/ppcp-settings/src/Data/CommonSettings.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Common settings class
+ *
+ * @package WooCommerce\PayPalCommerce\Settings\Data
+ */
+
+declare( strict_types = 1 );
+
+namespace WooCommerce\PayPalCommerce\Settings\Data;
+
+use RuntimeException;
+
+/**
+ * Class CommonSettings
+ *
+ * This class serves as a container for managing the common settings that
+ * are used and managed in various areas of the settings UI
+ *
+ * Those settings mainly describe connection details and are initially collected
+ * in the onboarding wizard, and also appear in the settings screen.
+ */
+class CommonSettings extends AbstractDataModel {
+
+	/**
+	 * Option key where profile details are stored.
+	 *
+	 * @var string
+	 */
+	protected const OPTION_KEY = 'woocommerce-ppcp-data-common';
+
+	/**
+	 * Get default values for the model.
+	 *
+	 * @return array
+	 */
+	protected function get_defaults() : array {
+		return array(
+			'use_sandbox'           => false,
+			'use_manual_connection' => false,
+			'client_id'             => '',
+			'client_secret'         => '',
+		);
+	}
+
+	// -----
```

Comment:
leftover?

Reply:
We can remove it anytime. My idea was to have a "separator" between regular class methods (top part), and getters/setters (bottom)

Reply:
sure, thanks for the explanation.